packages/core/threads/thingsboard_thread.py

```diff
@@ -0,0 +1,125 @@
+from .abstract_thread import AbstractThread
+from packages.core import types, utils, interfaces
+from tb_device_mqtt import TBDeviceMqttClient  # type: ignore
```

Comment:
Could you please inline the import of the `tb_device_mqtt` library. I've seen the code of that library and don't trust it yet.

Reply:
Can you give an example of what you mean?